src/annotator/integrations/html.ts

```diff
@@ -168,9 +168,17 @@ export class HTMLIntegration extends TinyEmitter implements Integration {
       this._deactivateSideBySide();
     }
     this._sideBySideActive = active;
+    this.container.classList.toggle(
+      'hypothesis-sidebyside-active',
+      this._sideBySideActive
+    );
```

Comment:
I moved this from the guest to here so that it will get updated if the side-by-side state changes in the integration. I think we may end up removing this soon though as the approach to side-by-side in the video player is going to change.